lib/ansible/modules/cloud/azure/AzureGuidelines.md

```diff
@@ -135,3 +141,14 @@ Samples should include all possible combinations of input parameters showing how
 - query all the resources of given type in the resource group
 - query specific resource
 - filter resources by tags
+
+## Testing
+
+Following rules shoudl apply to tests:
+- Do not create a separate test, add **azure_xx_facts** tests to **azure_xx** module tests
```

Comment:
Do not create a separate test, add **`azure_rm_xx_facts`** tests to **`/test/integration/targets/azure_rm_xx`** module tests